code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -19,36 +10,45 @@
 	description = "A well-known alcohol with a variety of applications."
 	color = "#404030" // rgb: 64, 64, 48
 	nutriment_factor = 0
-	var/boozepwr = 10 //lower numbers mean the booze will have an effect faster.
+	var/boozepwr = 65 //Higher numbers equal higher hardness, higher hardness equals more intense alcohol poisoning
+
+/*
+Boozepwr Chart
+Note that all higher effects of alcohol poisoning will inherit effects for smaller amounts (i.e. light poisoning inherts from slight poisoning)
+In addition, severe effects won't always trigger unless the drink is poisonously strong
+All effects don't start immediately, but rather get worse over time; the rate is affected by the imbiber's alcohol tolerance
+
+0: Non-alcoholic
+1-10: Barely classifiable as alcohol - occassional slurring
+11-20: Slight alcohol content - slurring
+21-30: Below average - imbiber begins to look slightly drunk
+31-40: Just below average - no unique effects
+41-50: Average - mild disorientation, imbiber begins to look drunk
+51-60: Just above average - disorientation, vomiting, imbiber begins to look heavily drunk
+61-70: Above average - small chance of blurry vision, imbiber begins to look smashed
+71-80: High alcohol content - blurry vision, imbiber completely shitfaced
+81-90: Extremely high alcohol content - heavy toxin damage, passing out
+91-100: Dangerously toxic - swift death
+*/
 
 /datum/reagent/consumable/ethanol/on_mob_life(mob/living/M)
-	M.jitteriness = max(M.jitteriness-5,0)
-	if(current_cycle >= boozepwr*0.5)
-		var/drunk_value = sqrt(volume*1000/boozepwr)
-		if(volume >= boozepwr*0.2)
-			if(M.slurring < drunk_value)
-				M.slurring += 4
-			M.Dizzy(drunk_value)
-		if(volume >= boozepwr*0.8)
-			if(M.confused < drunk_value)
-				M.confused += 3
-		if(volume >= boozepwr*3.8)
-			M.adjustToxLoss(1, 0)
-			. = 1
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		H.drunkenness = max((H.drunkenness + boozepwr * ALCOHOL_RATE), 0)
```

Comment:
I'm still a bit concerned about balance. A single full glass of whiskey giving you brain damage is a bit too much.

The model isn't realistic, any alcohol with boozepwr below 14 can never make you even tipsy no matter how much you drink. On the other side of the spectrum, drinking small sips of whisky slowly but regularly will have you ending up beyond shitfaced.

It might be a good idea to include the reagent volume in the drunkenness calculation, maybe make it use a Clamp so your drunkenness can never go above a certain value.